google-cloud-contrib/google-cloud-logging-logback/README.md

```diff
@@ -0,0 +1,127 @@
+Logback Appender for Google Cloud Logging
+=========================================
+
+Logback appender `com.google.cloud.logging.CloudLoggingAppender` for
+[Google Cloud Logging](https://cloud.google.com/logging/).
+
+This library allows you to use [Logback](https://logback.qos.ch/) configuration and the [Slf4j](https://www.slf4j.org/) interface to log via Google Cloud logging
+
+-  [Homepage](https://googlecloudplatform.github.io/google-cloud-java/)
+-  [API Documentation](https://googlecloudplatform.github.io/google-cloud-java/apidocs/index.html?com/google/cloud/logging/package-summary.html)
+
+> Note: This client is a work-in-progress, and may occasionally
+> make backwards-incompatible changes.
+
+Quickstart
+----------
+If you are using Maven, add this to your pom.xml file
+```xml
+<dependency>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-logging-logback</artifactId>
+  <version>0.17.1-alpha</version>
+</dependency>
+```
+If you are using Gradle, add this to your dependencies
+```Groovy
+compile 'com.google.cloud:google-cloud-logging-logback:0.17.1-alpha'
+```
+If you are using SBT, add this to your dependencies
+```Scala
+libraryDependencies += "com.google.cloud" % "google-cloud-logging-logback" % "0.17.1-alpha"
+```
+
+Usage 
+-----
+
+Add the appender to your [Logback configuration](https://logback.qos.ch/manual/configuration.html) `logback.xml`
+```xml
+<configuration>
+  <appender name="CLOUD" class="com.google.cloud.logging.contrib.LoggingAppender">
```

Comment:
It's certainly a matter of style, but I would probably pick `com.google.cloud.logging.logback.LoggingAppender`